src/components/tooltip/utils/mergeRefs.ts

```diff
@@ -0,0 +1,11 @@
+export function mergeRefs<T = any>(...refs: Array<React.MutableRefObject<T> | React.LegacyRef<T>>): React.Ref<T> {
+  return (value) => {
```

Comment:
요로캐 해보면 어떨까요

```suggestion
import { type MutableRefObject, type LegacyRef } from 'react';

export function mergeRefs<T = unknown>(...refs: Array<MutableRefObject<T> | LegacyRef<T>>): React.Ref<T> {
  return (value) => {
```

- unknown
요건 제 취향의 영역이긴합니다

- import from react
제가 난독증 중증 환자라 조금 줄여볼 수 있을 것 같아서 ,,,